src/cpu.rs

```diff
@@ -20,6 +22,14 @@ impl Core {
     pub fn is_active(&self) -> bool {
         *self == core()
     }
+
```

Comment:
Not necessary, remove.

Reply:
Didn't add this function, but I can remove it.

Reply:
Not `is_active`. We dont need the new one - `all` - that you've introduced.

Reply:
I used this function in my code to not repeat the same code. I can make it private oder move the function to my code.

Reply:
Isn't `EnumSet::all()` sufficient?

Reply:
If that's not clear, here's how you should call it: `EnumSet::<Core>::all()`.